README.md

```diff
@@ -31,6 +31,9 @@
 
 </p>
 
+**Warning**: Prefect 2, currently on the `orion` branch, will soon become the default package installed with `pip install prefect`. Flows written with Prefect 1 will require modifications to run with Prefect 2. Please ensure that your package management process enables you to make the transition when the time is right for you.
```

Comment:
Prefect 2 or 2.0 (and likewise 1 or 1.0)? We've been using the 2.0 usage, but I'm not married to it.

Reply:
Good point, I should keep in consistent.